config.json

```
@@ -210,6 +210,21 @@
         ],
         "status": "wip"
       },
+      {
+        "slug": "bird-watcher",
+        "name": "Bird Watcher",
+        "uuid": "bdb493de-b07d-4669-9890-5bebeddbf1e8",
+        "concepts": [ 
+          "for-loops", 
+          "increment-decrement"
```

```suggestion
          "for-loops"
```

The `increment-decrement` "concept" was actually included in `arithmetic-operations` so we should probably just delete it.